integration/metrics_test.go

```diff
@@ -153,10 +153,12 @@ log_fuse_operations = true
 
 	manipulateZtocMetadata := func(zt *ztoc.Ztoc) {
 		for i, md := range zt.FileMetadata {
-			md.UncompressedOffset += 2
-			md.UncompressedSize = math.MaxInt64
-			md.PAXHeaders = map[string]string{"foo": "bar"}
-			zt.FileMetadata[i] = md
+			if md.Type == "reg" {
```

Comment:
It would be good to include a comment here explaining the why only modify regular files similar to the explanation you added in the PR description.